data/tests/modules/etc/dnf/modules.d/base-runtime.module

```diff
@@ -2,5 +2,6 @@
 stream = f26
 version = 1
 profiles = 
-enabled = 1
-locked = 0
\ No newline at end of file
+locked = 0
```

Comment:
Please could you remove the change. It appears when dnf crashes during the test.